doc/api.rst

```diff
@@ -13,7 +13,17 @@ MNE BIDS (:py:mod:`mne_bids.mne_bids`):
 .. autosummary::
    :toctree: generated/
 
-   raw_to_bids
+   write_raw_bids
+   make_bids_filename
```

Comment:
the older API was approaching monolithic design. See [unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy#Eric_Raymond%E2%80%99s_17_Unix_Rules) for inspiration on monolithic vs. modular design.